### PR TITLE
tidb cloud: support decreasing the node size to scale in a Dedicated Tier

### PR DESCRIPTION
**Pull request by Oreoxmt**
### What is changed, added or deleted? (Required)

TiDB Cloud supports decreasing the size of TiDB, TiKV, and TiFlash nodes to scale in a Dedicated Tier.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tidb-cloud/scale-tidb-cluster.md [hunk 1]
```diff
@@ -99,19 +99,50 @@ You can also increase the storage of a TiKV or TiFlash node using TiDB Cloud API
...
+
+You can also increase the size of a TiDB, TiKV, or TiFlash node using TiDB Cloud API through the [Modify a Dedicated Tier cluster](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/UpdateCluster) endpoint. Currently, TiDB Cloud API is still in beta. For more information, see [TiDB Cloud API Documentation](https://docs.pingcap.com/tidbcloud/api/v1beta).
 
-To increase the node size, take the following steps:
+### Decrease node size
 
-1. In the TiDB Cloud console, navigate to the **Clusters** page for your project.
+> **Note:**
+>
+> - Decreasing node size is only available to clusters that are hosted on AWS and created after `{PLACEHOLDER}`.
```
Comment (SunRunAway) [inline tidb-cloud/scale-tidb-cluster.md:132]:
```suggestion
> - Decreasing node size is only available to clusters that are hosted on AWS and created after `2022/12/31`.
```

File: tidb-cloud/scale-tidb-cluster.md [hunk 2]
```diff
@@ -99,19 +99,50 @@ You can also increase the storage of a TiKV or TiFlash node using TiDB Cloud API
...
+
+You can also increase the size of a TiDB, TiKV, or TiFlash node using TiDB Cloud API through the [Modify a Dedicated Tier cluster](https://docs.pingcap.com/tidbcloud/api/v1beta#tag/Cluster/operation/UpdateCluster) endpoint. Currently, TiDB Cloud API is still in beta. For more information, see [TiDB Cloud API Documentation](https://docs.pingcap.com/tidbcloud/api/v1beta).
 
-To increase the node size, take the following steps:
+### Decrease node size
 
-1. In the TiDB Cloud console, navigate to the **Clusters** page for your project.
+> **Note:**
+>
+> - Decreasing node size is only available to clusters that are hosted on AWS and created after `2022/12/31`.
```
Comment (SunRunAway) [inline tidb-cloud/scale-tidb-cluster.md:132]:
```suggestion
> - Decreasing node size is only available to clusters that are hosted on AWS and created after 2022/12/31.
```

**Review by SunRunAway — APPROVED** [review]

File: tidb-cloud/scale-tidb-cluster.md [hunk 3]
```diff
@@ -99,19 +99,50 @@ You can also increase the storage of a TiKV or TiFlash node using TiDB Cloud API
 
 For a running cluster, AWS and Google Cloud do not allow in-place storage capacity downgrade.
 
-## Increase node size
+## Change node size
+
+You can change the size (including vCPUs and memory) of TiDB, TiKV, or TiFlash nodes.
+
+### Increase node size
```
Comment (ran-huang) [inline tidb-cloud/scale-tidb-cluster.md:106]: Is it possible to merge `### Increase node size` and `### Decrease node size`? They look basically the same.

Comment (ran-huang) [reply tidb-cloud/scale-tidb-cluster.md:106]: The only differences lie in the words `increase` and `decrease`.
image

Comment (csuzhangxc) [reply tidb-cloud/scale-tidb-cluster.md:106]: I think we can merge `increase` and `decrease`, @SunRunAway

Comment (SunRunAway) [reply tidb-cloud/scale-tidb-cluster.md:106]: I have no preference.

Comment (SunRunAway) [reply tidb-cloud/scale-tidb-cluster.md:106]: Also, there are "Increase node number" and "Decrease node number"  In "Change node number".  So if changes are to be made, the entire document will hopefully be consistent.

Comment (ran-huang) [reply tidb-cloud/scale-tidb-cluster.md:106]: "Decrease node number" comes with a warning on _insufficient storage space_. If we're to refactor the doc, please take notice of that warning. Rest LGTM

**Comment by Oreoxmt** [comment]
@ran-huang PTAL again

**Review by ran-huang — APPROVED** [review]

**Review by csuzhangxc — APPROVED** [review]

**Pull request merged**